dborutils/batch_manager.py

```diff
@@ -1,130 +0,0 @@
-from datetime import datetime
-from decimal import Decimal
-import logging
-logger = logging.getLogger(__name__)
-
-
-class BatchManager(object):
```

Comment:
This class still is imported in dborutils/copy_service.py, and is used there.  However, that module contains only four classes: `NoodleCopyService`, `NoodleMongoProdCopyService`, `NoodleScrapedMongoCopyService`, and `NoodleScrapedMongoListCopyService`, which are not used anywhere else.  Let's drop that module as well.